docs/source/playbooks.rst

```diff
@@ -131,7 +143,7 @@ and group variables files may help you organize your variable values more
 easily. Included with the sample playbook is a sample variables
 file `all.yml`_.
 
-The value for the property **BPXK_AUTOCVT** must be configured to ``ON``.
+The value for the property **BPXK_AUTOCVT** must be configured to ``'ON'``.
```

Comment:
I think the variable should be `_BPXK_AUTOCVT`

Reply:
Fixed good catch